utils/update-db.js

```diff
@@ -33,12 +33,14 @@ const SEMIVOICED_KATAKANA_TO_HIRAGANA = [
   0x3071, 0x3074, 0x3077, 0x307a, 0x307d
 ];
 
+// Since we only get the first letter, two letter codes need to processed
+// later.
 const SUPPORTED_REF_TYPES = new Set([
   'B',
   'H',
   'L',
   'E',
-  'D', // We'll manually filter this down to DK later
+  'D', // DK and DN
```

Comment:
Yeah, but since ref contains the type letter appended to the information itself (DN35 or B5) in order to generically do the checks I changed the logic to see if each reference was prefixed by one of these references. Given the short size of this list, the the performance change won't matter!